src/components/search/index.js

```diff
@@ -17,7 +17,7 @@ import { Root } from "./styles"
 import "./styles.css"
```

Comment:
we probably can remove a lot of `styles.css`

Reply:
we could remove them all.

Reply:
Unfortunately we can't remove them all because we can't style widget of instance search without global css class https://www.algolia.com/doc/guides/building-search-ui/widgets/customize-an-existing-widget/react/#style-your-widgets